Samples/NetworkMediaStreamer/RTPMediaStreamer/inc/RTPStreamSink.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for more information
+
+#pragma once
+
+constexpr BYTE h264payloadType = 96;
+constexpr size_t rtpHeaderSize = 12;
+constexpr size_t stapAtypeHeaderSize = 1;
+constexpr size_t stapASzHeaderSize = 2;
+class TxContext final
+{
+    uint16_t m_localRTPPort, m_localRTCPPort, m_remotePort;
+    sockaddr_in m_remoteAddr;
+    SOCKET m_RtpSocket, m_RtcpSocket;
+    winrt::PacketHandler m_packetHandler;
+
+public:
+    TxContext(std::string destination, winrt::PacketHandler packetHandler = nullptr);
+    ~TxContext();
+    void SendPacket(winrt::Windows::Storage::Streams::IBuffer buf);
+
+    uint32_t m_ssrc;
+    uint64_t m_u64StartTime;
+    uint32_t m_SequenceNumber;

```

Comment:
>S [](start = 15, length = 1)

nit: I notice that you sometime put a cap letter after the _m and sometimes not across most of your header files, try to stick to a consistent rule, i.e. non-capital letter for the first word or type prefix following m_